code/drasil-docLang/lib/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -120,45 +120,62 @@ assumpF otherContents = SRS.assumpt (assumpIntro : otherContents) []
        atStartNP' (the assumption), S "refine the", phrase scope,
        S "by providing more detail"]
 
+emptySectSentence :: NamedIdea n => n -> Contents
+emptySectSentence var = foldlSP[ S "There are no" +:+ plural var]
+
 -- | Wrapper for 'thModelIntro'. Takes the program name and other 'Contents'.
 thModF :: (Idea a) => a -> [Contents] -> Section
-thModF progName otherContents = SRS.thModel (thModIntro progName : otherContents) []
+thModF _        []                   = SRS.thModel [thModIntroNoContent] []
```

Comment:
```suggestion
thModF _        []            = SRS.thModel [thModIntroNoContent] []
```

Reply:
I will make that change right now. Thanks for catching that!